lib/client.js

```diff
@@ -73,6 +74,17 @@ function MatrixClient(opts) {
     this._syncingRooms = {
         // room_id: Promise
     };
+    this.callList = {
+        // callId: MatrixCall
+    };
+
+    // try constructing a MatrixCall to see if we are running in an environment
+    // which has WebRTC. If we are, listen for and handle m.call.* events.
+    var call = webRtcCall.createNewMatrixCall(this);
```

Comment:
In angular, I ended up making this return a dummy object if the browser had no WebRTC. This makes it easier for the client to still manage some about of call flow (ie. ringing, stop ringing when other party hangs up or call is answered elsewhere).